### ISSUE TITLE
Accessibility link doesn't lead anywhere

### ISSUE THREAD

**Issue by Bodacious** [DMPRoadmap/roadmap#1734]
Please complete the following fields as applicable:

**Expected behaviour:**
When I click the accessibility link, it should take me to a phase related to accessibility.

**Actual behaviour:**
Nothing happens

**Steps to reproduce:**
Click on the "Accessibility" link at the foot of the page

**Comment by sjDCC** [comment]
This should just be a link out to an accessibility page that's added per installation as needed. It doesn't work on roadmap-stage but on DMPTool it redirects to https://www.cdlib.org/about/accessibility.html Each deployment should set or hide the link.

Suspect this is fine to close out, unless there's an issue with the code

**Comment by Bodacious** [comment]
@sjDCC Any idea how this is customised?

As far as I can see from the code, the `href` attribute for the link is hard-coded, and doesn't seem to support customisation within the template code here:

https://github.com/DMPRoadmap/roadmap/blob/master/app/views/layouts/_footer.html.erb#L17

@briri any ideas?

**Comment by sjDCC** [comment]
I assumed it was part of the yaml rather than hard-coded but maybe not…

**Comment by briri** [comment]
I just looked through the commit history and it looks like its always been a dead link on the core codebase. The DMPTool links out to our main CDL accessibility statement page I'm not sure about Tuuli or DMPonline.

Maybe you can suggest a default accessibility statement to include in the core codebase @sjDCC and @stephaniesimms? Something generic enough for brand new installations

**Comment by sjDCC** [comment]
We don't have an accessibility link on Tuuli or DMPonline. Perhaps remove from core codebase or add something generic like you say

**Comment by briri** [comment]
Ah yeah, if that's the case I would say go ahead and delete it @Bodacious
